ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -0,0 +1,897 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.daml.platform.store.appendonlydao
+
+import java.sql.Connection
+import java.time.Instant
+import java.util.Date
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import anorm.SqlParser._
+import anorm.{Macro, RowParser, SQL, SqlParser}
+import com.daml.daml_lf_dev.DamlLf.Archive
+import com.daml.ledger.api.domain
+import com.daml.ledger.api.domain.{LedgerId, ParticipantId, PartyDetails}
+import com.daml.ledger.api.health.HealthStatus
+import com.daml.ledger.participant.state.index.v2.{
+  CommandDeduplicationDuplicate,
+  CommandDeduplicationNew,
+  CommandDeduplicationResult,
+  PackageDetails,
+}
+import com.daml.ledger.participant.state.v1._
+import com.daml.ledger.resources.ResourceOwner
+import com.daml.ledger.{TransactionId, WorkflowId}
+import com.daml.lf.archive.Decode
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.{PackageId, Party}
+import com.daml.lf.engine.ValueEnricher
+import com.daml.lf.transaction.{BlindingInfo, GlobalKey}
+import com.daml.lf.value.Value.ContractId
+import com.daml.logging.LoggingContext.withEnrichedLoggingContext
+import com.daml.logging.{ContextualizedLogger, LoggingContext}
+import com.daml.metrics.{Metrics, Timed}
+import com.daml.platform.configuration.ServerRole
+import com.daml.platform.indexer.OffsetStep
+import com.daml.platform.store.Conversions._
+import com.daml.platform.store.SimpleSqlAsVectorOf.SimpleSqlAsVectorOf
+import com.daml.platform.store._
+import com.daml.platform.store.appendonlydao.CommandCompletionsTable.prepareCompletionsDelete
+import com.daml.platform.store.appendonlydao.events._
+import com.daml.platform.store.dao.events.EventsTableDelete
```

Comment:
this should probably be `appendonlydao`. There is a version of that class in the new hirerachy

Reply:
sharp eye! changed